class/defaults.yml

```
@@ -1,7 +1,11 @@
 parameters:
   openshift4_authentication:
     namespace: openshift-config
-    sudoGroupName: sudoers
+    # Deprecated: add name to 'sudoGroups' instead.
+    sudoGroupName: null
+    sudoGroups:
+      - sudoers
```

Thinking about it again, we may want to leave `sudoGroups` empty by default. With the new implementation this should create valid rolebindings with empty `subjects` arrays using the adjusted default config.

As it is currently, naive use of the new parameter will create rolebindings for a nonexistent group `sudoers`. An adversary with permissions to create groups could create this group to elevate themselves to full cluster-admin.

```suggestion
    sudoGroups: []
```

---

You are right. This would make it a breaking technically. But we already oveerride the paramter in global defaults anyway.